Program.cs

```diff
@@ -3,17 +3,48 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to sort and swap special characters.
+// Given a char array, a special character S and sort order (default order Ascending), print sorted array keeping elements matching S at the last.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Sort and Swap Array</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method gets valid user input and displays the sorted and swapped array</summary>
+      static void Main () {
+         char[] letters;
+         string splCharInput;
+         for (; ; ) {
+            Console.Write ("Enter the characters (only alphabets) to be sorted (as string): ");
+            letters = Console.ReadLine ().ToLower ().ToCharArray ();
+            Console.Write ("Enter the special character to be swapped to the end: ");
+            splCharInput = Console.ReadLine ();
+            if (letters.Length == 0
+                   || letters.Any (x => !char.IsLetter (x))
+                   || splCharInput.Length != 1
+                   || splCharInput.Any (x => !char.IsLetter (x))) {
+               Console.Write ("Invalid Input.\n");
+               continue;
+            }

```

Comment:
This is not the correct way to validate the input values. The first input may be valid, and the second input may be invalid. But with the above code, the user will have to type-in the first input again:
<img width="566" alt="image" src="https://github.com/RaaneshKumar/Training/assets/75829870/4772e753-10ef-4e85-9389-3f0e4511325e">